lib/flippant/adapters/postgres.rb

```diff
@@ -140,7 +139,7 @@ def setup
       private
 
       def conn
-        pool.with_connection do |connection|
+        ActiveRecord::Base.connection_pool.with_connection do |connection|
```

Comment:
It would be nice to have `pool` be a method that returns this rather than chaining the methods here.